cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -16,6 +16,8 @@ You can also find the URL to the latest binaries from the {product-title} web co
 
 include::modules/developer-cli-odo-installing-odo-on-linux.adoc[leveloffset=+1]
 
+include::modules/developer-cli-odo-installing-odo-on-linux-on-ppc64le.adoc[leveloffset=+1]
```

Comment:
To keep this consistent with Silke's naming convention for IBM Z, should we not be naming this module: developer-cli-odo-installing-odo-on-linux-on-IBM-Power.adoc?

Reply:
Sure thing!